src/Shouldly/ChuckedAWobbly.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace Shouldly
 {
+    [Obsolete]
```

Comment:
Maybe add a comment in to say it has been left for compability reasons. The other thing which makes me nervous about removing this when/if we next bump major is that it's internal, so people would be accessing via reflection if they are using it and will not receive build warnings.